### PR TITLE
Sprint7 Add Games in 'Find Match' tab functionality

### PR DESCRIPTION
**Pull request by dnguyen3031**

### CODE REVIEW

File: backend.py [hunk 1]
```diff
@@ -316,6 +316,29 @@ def add_to_queue():
...
+        user = User({"_id": user_id})
+        search_game = Game().find_by_name(game_name)[0]
+        if search_game: #actually returns a game
+            if user.reload():
+                user["games_table"][game_name] = {
+                            "game_score": 400,
+                            "time_played": 0
+                        }
+                user["_id"] = ObjectId(user_id)
+                print("The code made it here LOLOLOL")
```
Comment (xhwallace) [inline backend.py:333]: remove print statement

File: backend.py [hunk 2]
```diff
@@ -316,6 +316,29 @@ def add_to_queue():
...
+def add_new_game():
+    if request.method == 'PATCH':
+        game_name = request.args.get('game_name')
+        user_id = request.args.get('id')
+        user = User({"_id": user_id})
+        search_game = Game().find_by_name(game_name)[0]
+        if search_game: #actually returns a game
+            if user.reload():
+                user["games_table"][game_name] = {
+                            "game_score": 400,
```
Comment (xhwallace) [inline backend.py:329]: 400 is a magic number. Make it clear that this is the initial elo. The initial ELO for new users is 1000, so this shouls also be 1000.

**Review by xhwallace — COMMENTED** [review]
It looks like this code doesn't check if the user already has the game as part of their games table. If so, they should not be able to reset their elo by adding a game they already have. 
Otherwise this is pretty bare bones (no user feedback) but functional.

**Review by prcox — APPROVED** [review]
I don't see any problems that xander didn't catch

**Comment by jlu51** [comment]
Everything looks good

**Comment by jlu51** [comment]
Everything looks good to me!

**Review by mchris25 — APPROVED** [review]
Changes look good to me. Runs with the current build so its OK

**Pull request merged**